assets/home/header.js

```diff
@@ -3,10 +3,20 @@
  */
 import { __ } from '@wordpress/i18n';
 
+/**
+ * Internal dependencies
+ */
+import LogoTree from '../icons/logo-tree.svg';
+import SenseiLogoType from '../icons/sensei-logo-type.svg';
+
 const Header = () => (
 	<header>
-		<h1 className="wp-heading-inline">
-			{ __( 'Sensei Home', 'sensei-lms' ) }
+		<h1 className="wp-heading-inline sensei-home-title">
+			<span className="screen-reader-text">
+				{ __( 'Sensei', 'sensei-lms' ) }
```

Comment:
Maybe call "Sensei Home" instead of just "Sensei"?

```suggestion
				{ __( 'Sensei Home', 'sensei-lms' ) }
```

Reply:
I think probably Sensei is better because it's what we're displaying to the user. Make sense?

Reply:
hmmm, good point. I mentioned to put as Sensei Home because it's the page we're displaying to the user, but actually I think either should be valid. :)